packages/main/src/TableGroupRow.js

```diff
@@ -150,6 +150,10 @@ class TableGroupRow extends UI5Element {
 		this._colSpan = this.visibleColCount();
 	}
 
+	_onfocusin(event) {
```

Comment:
You should not access the table via `parentElement`. This is not always guaranteed to work (with transitive slotting the parent will be a slot) and is an antipattern. If the row must tell the table to do something, it must fire an event and the table must be listening to this event and react on its own.

In general, parent -> child communication (top-down communication) is via setting properties/calling methods of children. However, bottom-up communication should only be done by events.

Reply:
Implemented as part of #4698.